'yarn start ' and 'npm start'  failing while loading web UI

**Describe the bug**
'yarn start' is giving errors while trying to run the web UI  in localhost.Trying to run BentoML Web UI.Followed the section 'How to run and develop BentoML Web UI from 'https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md to load the UI.
After running the yarn and npm run build commands as directed its throwing errors and UI is not getting loaded after 'npm start' and 'yarn start'

**To Reproduce**
Steps to reproduce the behavior:
1. Go to Go to 'https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md'
2. Scroll down to 'How to run and develop BentoML Web UI'.'
3. See error after 'yarn install' and 'yarn start' command.
   yarn run v1.22.4
$ node dist/index.js
internal/modules/cjs/loader.js:716
    throw err;
    ^

Error: Cannot find module '/home/techmadmin/Documents/BentoML/bentoml/yatai/web/dist/index.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:713:15)
    at Function.Module._load (internal/modules/cjs/loader.js:618:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:931:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.


**Expected behavior**
WebUI served on Localhost

**Screenshots/Logs**
![bentouiyarnstart](https://user-images.githubusercontent.com/10370721/87624209-c846ad00-c744-11ea-8971-099c94e603e7.png)

Environment:

OS: Ubuntu 18.04

Python/BentoML Version [ Python 3.7.1, BentoML-0.8.2]

Node version -v12.8.0
-Yarn Version - 1.22.

--- a/bentoml/yatai/web/package.json
+++ b/bentoml/yatai/web/package.json
@@ -5,7 +5,7 @@
   "description": "",
   "license": "Apache-2.0",
   "scripts": {
-    "start": "node dist/index.js",
+    "start": "node dist/bundle.js",
     "dev": "bash -c 'concurrently \"nodemon -- ${0} ${1} ${2}\" \"npm run watch:client\"'",
     "watch": "concurrently \"npm:watch:*\"",
     "watch:server": "nodemon",

--- a/bentoml/yatai/web/src/server.ts
+++ b/bentoml/yatai/web/src/server.ts
@@ -10,18 +10,19 @@ const logger = getLogger();
 
 const createRoutes = (app, yataiClient) => {
   app.get("/api/ListBento", async (req: Request, res: Response) => {
+    const requestQuery: any = Object.assign({}, req.query);
     if (req.query.limit && typeof req.query.limit == "string") {
-      req.query.limit = Number(req.query.limit);
+      requestQuery.limit = Number(req.query.limit);
     }
     if (req.query.offset && typeof req.query.offset == "string") {
-      req.query.offset = Number(req.query.offset);
+      requestQuery.offset = Number(req.query.offset);
     }
-    let verifyError = bentoml.ListBentoRequest.verify(req.query);
+    let verifyError = bentoml.ListBentoRequest.verify(requestQuery);
     if (verifyError) {
       logger.error({ request: "ListBento", error: verifyError });
       return res.status(400).json({ error: verifyError });
     }
-    let requestMessage = bentoml.ListBentoRequest.create(req.query);
+    let requestMessage = bentoml.ListBentoRequest.create(requestQuery);
     try {
       const result = await yataiClient.listBento(requestMessage);
       logger.info({
@@ -88,15 +89,16 @@ const createRoutes = (app, yataiClient) => {
   });
 
   app.get("/api/ListDeployments", async (req: Request, res: Response) => {
+    const requestQuery :any = Object.assign({}, req.query);
     if (req.query.limit && typeof req.query.limit == "string") {
-      req.query.limit = Number(req.query.limit);
+      requestQuery.limit = Number(req.query.limit);
     }
-    let verifyError = bentoml.ListDeploymentsRequest.verify(req.query);
+    let verifyError = bentoml.ListDeploymentsRequest.verify(requestQuery);
     if (verifyError) {
       logger.error({ request: "ListDeployments", error: verifyError });
       return res.status(400).json({ error: verifyError });
     }
-    let requestMessage = bentoml.ListDeploymentsRequest.create(req.query);
+    let requestMessage = bentoml.ListDeploymentsRequest.create(requestQuery);
     try {
       const result = await yataiClient.listDeployments(requestMessage);
       logger.info({

